website/content/commands/install.mdx

```diff
@@ -79,15 +79,16 @@ and disable the UI, the command would be:
 
 #### kubernetes Options
 
+- `-k8s-config-path=<string>` - Path to the kubeconfig file to use,
 - `-k8s-advertise-internal` - Advertise the internal service address rather than the external. This is useful if all your deployments will be able to access the private service address. This will default to false but will be automatically set to true if the external host is detected to be localhost.
 - `-k8s-annotate-service=<key=value>` - Annotations for the Service generated.
 - `-k8s-context=<string>` - The Kubernetes context to install the Waypoint server to. If left unset, Waypoint will use the current Kubernetes context.
+- `-k8s-helm-version=<string>` - The version of the Helm chart to use for the Waypoint runner install.The required version number format is: 'vX.Y.Z'.
 - `-k8s-cpu-request=<string>` - Configures the requested CPU amount for the Waypoint server in Kubernetes.
 - `-k8s-mem-request=<string>` - Configures the requested memory amount for the Waypoint server in Kubernetes.
 - `-k8s-cpu-limit=<string>` - Configures the CPU limit for the Waypoint server in Kubernetes.
 - `-k8s-mem-limit=<string>` - Configures the memory limit for the Waypoint server in Kubernetes.
 - `-k8s-namespace=<string>` - Namespace to install the Waypoint server into for Kubernetes.
-- `-k8s-openshift` - Enables installing the Waypoint server on Kubernetes on Red Hat OpenShift. If set, auto-configures the installation.
```

Comment:
The `waypoint-helm` chart (currently) has no configs related to Openshift, and since we're installing the server and runner with the Helm chart as of this PR, this config is no longer needed.